pkg/xray/resource_xray_license_policy.go

```diff
@@ -68,14 +69,16 @@ func resourceXrayLicensePolicyV2() *schema.Resource {
 										Type:     schema.TypeList,
 										Optional: true,
 										Elem: &schema.Schema{
-											Type: schema.TypeString,
+											Type:         schema.TypeString,
```

Comment:
Also, could you extract them as variables and then add them - it would make this whole structure flatter and easier to read

Reply:
will do later